app/assets/stylesheets/provider/_links.scss

```
@@ -23,6 +23,10 @@ table button.action {
   display: inline-block;
 }
 
+table a.ui-sortable-handler {
```

```suggestion
.ui-sortable-handler {
```

and if we choose it to no longer be `a` element it belongs in a different stylsheet.